client/src/components/pages/RobotInteractionCockpit/RobotInteractionCockpit.jsx

```diff
@@ -120,6 +183,55 @@ const RobotInteractionCockpit = (match) => {
                 </Button>
               </>
             )}
+          {currentStep !== 0 && (
+            <>
+              <Row>
+                <Col  xs={24} lg={24} xl={12}>
+                  <Title
+                    style={{
+                      marginBottom: '0px',
+                      marginLeft: '10px',
+                    }}
+                    level={5}
+                  >
+                    Robot Status
+                  </Title>
+                  <Card style={{ margin: '10px' }} hoverable>
+                    <Row>
+                      <Col span={18}>
+                        <Title
+                          style={{
+                            marginBottom: '0px',
```

Comment:
As far as I know, we decided to never have more than 2 styles in a tag without refactoring it into a css file

Reply:
fixed